platform/darwin/src/MGLSource_Private.h

```diff
@@ -13,11 +15,18 @@ namespace mbgl {
 @interface MGLSource (Private)
 
 /**
- Initializes and returns a source with a raw pointer to the backing store.
+ Initializes and returns a source with a raw pointer to the backing store,
+ associated with a style.
  */
 - (instancetype)initWithRawSource:(mbgl::style::Source *)rawSource;
 
 /**
+ Initializes and returns a source with an owning pointer to the backing store,
+ unassociated from a style.
+ */
+- (instancetype)initWithPendingSource:(std::unique_ptr<mbgl::style::Source>)pendingSource;
```

Comment:
This method doesn’t appear to be implemented in MGLSource.mm. Calling it will result in a crash.